source/troubleshooting.txt

```diff
@@ -89,8 +89,52 @@ to create new projects.
 
 The user's IP address or API key that you use to authenticate is not on
 the :atlas:`access list </configure-api-access>` for the requested
-project. Add your IP address or API key to the
-:atlas:`access list </configure-api-access>` to run commands.
+project. Add your IP address or API key to the :atlas:`access list 
+</configure-api-access>` to run commands. To learn more, see 
+:atlas:`Get Started with the Atlas Administration API 
+</configure-api-access>`.
```

Comment:
```suggestion
project. Add your IP address to the :atlas: access list (link to a guide) or to the API access list (link to a guide) to run commands.
```

Reply:
This Get Started Link  does not cover API access list and this is what customers facing the problem will reach before reading the section below. This is why I think we should rethink this
```
Get Started with the Atlas Administration API 
</configure-api-access>
```

Reply:
@andreaangiolillo, this page is exactly where I pulled the steps from. I added a list with two links (project, API). I linked to the top-level so that they can go here for comprehensive info. Otherwise, the exact steps are now pulled into this section.